block/stream/record_file_item.proto

```diff
@@ -82,11 +83,29 @@ message RecordFileItem {
      * The contents of sidecar files for this block.<br/>
      * Each block can have zero or more sidecar files.
      */
-    repeated bytes sidecar_file_contents = 3;
+    repeated proto.SidecarFile sidecar_file_contents = 3;
 
     /**
      * A collection of RSA signatures from consensus nodes.<br/>
      * These signatures validate the hash of the record_file_contents field.
      */
-    repeated bytes record_file_hash_signatures = 4;
+    repeated RecordFileSignature record_file_signatures = 4;
+}
+
+/**
+ * A signature by a node on the SHA384 hash of the record file.
+ */
+message RecordFileSignature {
+    /**
+     * A single RSA signature.<br/>
+     * This is the RSA signature of the node on the SHA384 hash of
+     * the record file
+     */
+    bytes signatures_bytes = 1;
+
+    /**
+     * A unique node identifier.<br/>
+     * This is the node id of the consensus node that created this signature.
+     */
+    int32 node_id = 2;
```

Comment:
`NodeAddress.nodeId` is of type int64, so for consistency, better to stick to the same type here?

Reply:
Not necessary, as node IDs are only 0-31 and once we turn on block streams, this will never be used thereafter.
So this value will only ever be much much smaller than int32.

Reply:
sounds good